src/nlp/utils/mock_download_manager.py

```diff
@@ -0,0 +1,136 @@
+# coding=utf-8
```

Comment:
move the `MockDownloadManager` to the actualy `src/nlp` code base since it is used in one of the commands now - is that fine for you @lhoestq @thomwolf ?